concepticondata/conceptlists.tsv

```diff
@@ -336,3 +336,5 @@ Mountain-1978-267	Mountain, Kathy	1978		267	areal	Spanish	Tucanoan languages		Mo
 Chacon-2019-220	Chacon, Thiago Costa and Gonçalves, Artur Garcia and da Silva, Lucas Ferreira	2019		243	basic, areal	Portuguese	Arawakan languages	https://doi.org/10.15446/fyf.v32n2.80814	Chacon2019		A 243-item list (note that the list actually contains 243 items in contrast to the 220 stated in the paper) used as the basis for a comparative and dialectogical analysis of Arawak languages and its varieties from the Upper Rio Negro in Brazil. The data are based on documents created by missionaries in the 1950s. 		
 Rowe-1954-232	Rowe, John Howland	1954		232	areal	Spanish	South American languages		Rowe1954b	Rowe1954b	This list was used for a comparison of native South American languages and for exploring their potential relatedness; the items include basic concepts as well as areally specific concepts from the natural domain. It was in part inspired by the works of Curt Nimuendajú and has in turn given rise to a larger concept list termed Swadesh-Rowe which was put together and used mainly by SIL linguists working on South American languages.	137-146	
+Johansson-2020-344	Johansson, Niklas Erben and Anikin, Andrey and Carling, Gerd and Holmer, Arthur	2020		344	basic	English	Global	https://doi.org/10.1515/lingty-2020-2034	Johansson2020	Johansson2020	The given 344-item concept list underlies the analysis of sound symbolic patterns across 245 language families. In comparison to the previous attempts to establish sound-meaning associations, the current study benefits from a larger number of analyzed lexemes, improved transription systems and a sample of unrelated language families to avoid genealogical bias. The items for the concept list were selected based on their near-universality. They include the 56 oppositional concepts from [Johansson (2017)](:ref:Johansson-2017-56), the 64 kinship terms two steps from the ego with relative age distinctions  while the rest of the data come from Swadesh’s lists of [100](:ref:Swadesh-1971-100) and 207 items (1971) (the latter is likely to be the merged version of Swadesh's [100-item](:ref:Swadesh-1955-100) and [200-item](:ref:Swadesh-1952-200) lists), [the Leipzig-Jakarta list (Haspelmath & Tadmor 2009)](:ref:Tadmor-2009-100) and [Goddard and Wierzbicka’s (2002)](:bib:Goddard2002).	262-263	
```

Comment:
They refer to Swadesh 1971 (Swadesh, Morris. 1971. The origin and diversification of language. Edited post mortem by Joel
Sherzer. London: Transaction Publishers) for both the 100-item list and the 200-item list.

The source for Goddard's data cited in Johansson is different from those available in Concepticon (Goddard-2001-42 and Goddard-2011-63). I've managed to find it, though, and will send it to you per email. Here's an abstract from the review by George van Driem:



> In the six language-specific studies, each author attempts to identify categories of meaning in
>  individual languages that can ostensibly be taken to be exponents of primes. CLIFF GODDARD
>  claims to find 59 such meanings in colloquial Malay; CATHERINE TRAVIS, 59 in Colombian
>  Spanish; HILARY CHAPPELL, 59 in Mandarin Chinese; ROBERT BUGENHAGEN, 56 in Mangaaba-
>  Mbula a.k.a. Mangap-Mbula (an Austronesian language spoken by approximately 3,500 people
>  on Umboi Island in Papua New Guinea); ANNA WIERZBICKA, 60 in Polish; and NICK ENFIELD,
>  60 in Lao.

It could be the case that Johansson took a number of concepts, present in each of the prime lists.